plugin/pkg/admission/network/defaultingressclass/admission.go

```diff
@@ -141,10 +142,14 @@ func getDefaultClass(lister networkingv1listers.IngressClassLister) (*networking
 	if len(defaultClasses) == 0 {
 		return nil, nil
 	}
-
+	sort.Slice(defaultClasses, func(i, j int) bool {
+		if defaultClasses[i].CreationTimestamp.UnixNano() == defaultClasses[j].CreationTimestamp.UnixNano() {
+			return defaultClasses[i].Name < defaultClasses[j].Name
+		}
```

Comment:
this test cases is missing, `two defaults same timestamp, choose the one with the lower name`

Reply:
I think [this test](https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/network/defaultingressclass/admission_test.go#L141) has already done it.It has two deault classes with createTime equal to 0. I have updated `expectedClass` field in the commit. I will also update the test name.